hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/index/bloom/HoodieMetadataBloomIndexCheckFunction.java

```diff
@@ -113,7 +110,7 @@ protected void start() {
       }
 
       List<Pair<String, String>> partitionNameFileNameList = new ArrayList<>(fileToKeysMap.keySet());
-      Map<Pair<String, String>, ByteBuffer> fileToBloomFilterMap =
+      Map<Pair<String, String>, BloomFilter> fileToBloomFilterMap =
```

Comment:
Is my understanding correct that change from ByteBuffer to BloomFilter is only to avoid repetitive decoding? Is there any other reason?